Update payments-api-overview.md (#1168)
bigcommerce/dev-docs: docs/api-docs/payments/payments-api-overview.md

A declined payment will return a 4XX error with details if available.

### Authorization

> If you configure a payment gateway for authorization only, authorization happens at the time of processing. You will need to capture the order later through the control panel or via the Capture API. If you configure a payment gateway for authorization and capture, the payment will be authorized and captured at the time of processing.

Reviewer: ```suggestion
If you configure a payment gateway for authorization only, authorization happens at the time of processing. You will need to capture the order later using the control panel or the Capture API. If you configure a payment gateway for authorization and capture, the payment will be authorized and captured at the time of processing.
```

**Can I issue a refund?**

> You can issue a refund using the [Control Panel](https://support.bigcommerce.com/s/article/Processing-Refunds), the Refunds API, or through the payment gateway directly.

Reviewer: ```suggestion
You can issue a refund using the [control panel](https://support.bigcommerce.com/s/article/Processing-Refunds), the Refunds API, or through the payment gateway directly.
```

**How do I process payment for a capture credit card?**

> Once you have an authorized payment, perform the capture step using the [Control Panel](https://support.bigcommerce.com/s/article/How-can-I-set-my-payment-gateway-to-only-authorize-transactions-and-not-capture-the-funds-automatically) or the Capture API.

Reviewer: ```suggestion
Once you have an authorized payment, perform the capture step using the [control panel](https://support.bigcommerce.com/s/article/How-can-I-set-my-payment-gateway-to-only-authorize-transactions-and-not-capture-the-funds-automatically) or the Capture API.
```
